amundsen_application/static/css/_variables-default.scss

```diff
@@ -92,6 +92,10 @@ $icon-bg-brand: $brand-color-3 !default;
 $icon-bg-dark: $gray60 !default;
 $icon-bg-disabled: $gray20 !default;
 $red-triangle-warning: $sunset60;
+$icon-status-color: #f2f2f2;
+$icon-status-missed: $sunset60;
+$icon-status-hit: #00824C;
+$icon-size: 14px;
```

Comment:
I wouldn't bring this variables up here, it could be deceiving.
I would let them on the specific file.